spec/OpenLineage.md

```diff
@@ -133,7 +133,7 @@ Example of valid name is `BigQueryStatisticsJobFacet` and it's key `bigQuery_sta
 
 - **version**: Captures the dataset version when versioning is defined by database (ex. Iceberg snapshot ID)
 
-- **columnLineage**: Captures the column-level lineage
+- [**columnLineage**](facets/ColumnLineageDatasetFacet.md): Captures the column-level lineage
```

Comment:
`**[columnLineage](facets/ColumnLineageDatasetFacet.md)** :`
might be correct.

Reply:
this looks like it works? https://github.com/OpenLineage/OpenLineage/blob/ff0d87d30ed6c9fe39472788948266a6d3190585/spec/OpenLineage.md#dataset-facets

Reply:
> this looks like it works? https://github.com/OpenLineage/OpenLineage/blob/ff0d87d30ed6c9fe39472788948266a6d3190585/spec/OpenLineage.md#dataset-facets

I tried your format on number of different markdown editors, and they all seem to work!! So, I guess it's perfectly fine!